rwengine/src/render/OpenGLRenderer.cpp

```diff
@@ -468,3 +468,5 @@ const Renderer::ProfileInfo& OpenGLRenderer::popDebugGroup() {
 #endif
     return profileInfo[0];
 }
+
+Renderer::ShaderProgram::~ShaderProgram() = default;
```

Comment:
You already added the pure dtor in the header, what's the meaning behind this?

Reply:
It's for generating dtor implementation. If you want I can replace with empty implementation.